pipelines/templates/api-build.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           checkCoverage: true
           coverageFailOption: 'fixed'
           coverageType: 'lines'
-          coverageThreshold: '25'
+          coverageThreshold: '20'
```

Comment:
(minor) I would prefer decoupling the infra (ci/cd/etc...) from this feature changes, because this PR had a different intention. NBD though.